src/components/PBSCTextField.js

```diff
@@ -84,7 +85,7 @@ const PBSCTextField = (props) => {
         />
       );
     } else {
```

Comment:
maybe we could have another else if statement and put the return null as last:

else if(customIcon) {
return <TextInput.Icon icon={() => customIcon()} style={{ marginTop: 16 }}/>
}

return null;